### ISSUE TITLE
Cloud Object Storage

### ISSUE THREAD

**Issue by nelsonic** [dwyl/technology-stack#63]
We need to decide where to place our "bet" for cloud storage.
AWS S3 is one of the most expensive Object Storage Options.
BackBlaze is _by far_ the cheapest: 
https://www.backblaze.com/b2/cloud-storage-pricing.html
![image](https://user-images.githubusercontent.com/194400/46398398-f0d6f180-c6ec-11e8-8679-41c7c5f844ab.png)
According to this post on Reddit:
https://www.reddit.com/r/DataHoarder/comments/877z0s/options_for_uk_or_eu_based_cloud_storage_similar
BackBlaze are planning to open a Data Centre in Europe in Q3 of 2018 ...
If that happens we can use "B2" for all our backups and save 75% of the cost over s3! 😮

**Comment by matrixik** [comment]
http://gaul.org/object-store-comparison/

**Comment by matrixik** [comment]
You need to take into account that some of this services have some catches that are not described on this page (like Wasabi want you to pay up front for 90 days of storage for every file if I remember correctly).

**Comment by nelsonic** [comment]
@matrixik yeah, Wasabi are catering to people who can plan/forecast their usage requirements.
To be fair, if people don't have at least _some_ forecast-ability, they don't really "understand" their App.

https://www.scaleway.com are not _yet_ in the comparison table , because they haven't released their **Object Store**, but I suspect that their prices are going to be pretty competitive once they release!
Considering the price of their SSD-based Virtual Private servers and even "Bare Metal":
![image](https://user-images.githubusercontent.com/194400/46409729-8171f980-c70e-11e8-975d-a23b45a371c9.png)
https://www.scaleway.com/baremetal-cloud-servers/
![image](https://user-images.githubusercontent.com/194400/46409930-1674f280-c70f-11e8-8dd6-21b845db0887.png)

I'm thinking of using https://www.scaleway.com for my "pet project" 
(_which has no revenue so costs need to be kept as low as possible..._)

**Comment by matrixik** [comment]
https://blog.online.net/2018/10/04/scaleway-object-storage-beta/

**Comment by nelsonic** [comment]
@matrixik Thanks for sharing! 🥇 
Looks like the _bandwidth_ is where they will be making up for their "_cheap_" storage price:
![scaleway-object-storage-bandwidth-cost](https://user-images.githubusercontent.com/194400/46664586-5594bf00-cbb9-11e8-9844-8e7cfbc1ffab.png)

We may need to do some "arbitrage" when it comes to our object storage costs ... 🤔 
but for now, I think Scaleway looks to be the _best_ option for storing large volumes of arbitrary data. 👍 
Thanks again! ✨

**Comment by matrixik** [comment]
BackBlaze and C14 still looks cheaper.

**Comment by nelsonic** [comment]
@matrixik yeah, our only issue with BackBlaze is _location_ ... 🌎 
If they _open_ their EU-based data centre we can comply with EU-only storage laws ⚖️ 
for some of our more "_demanding_" clients which have _forced_ us to use AWS S3 until now. 💸

**Comment by nelsonic** [comment]
Meanwhile  ... since the last comment in this thread, `BackBlaze` has a datacenter in `Amsterdam`: 
https://help.backblaze.com/hc/en-us/articles/360034360834-EU-Region-FAQ 🇳🇱
This is _perfect_ for us to stay `EU GDPR` compliant. 👌
